engine/execution/computation/manager.go

```diff
@@ -41,7 +41,7 @@ const (
 	ReusableCadenceRuntimePoolSize = 1000
 )
 
-var uploadEnabled = true
+var uploadEnabled = false
```

Comment:
I think we can get rid of this variable by checking whether `len(e.uploaders) > 0`. If uploaded is enabled, there must be at least one `e.uploaders` .

Reply:
That flag is there to support toggling uploads from the admin server